### PR TITLE
Issue 574, order of precedence

### PR DESCRIPTION
**Pull request by wseltzer**
https://github.com/w3c/w3process/issues/574

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/w3process/pull/599.html" title="Last updated on Feb 15, 2022, 3:41 PM UTC (5833fe1)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/w3process/599/5c624a0...5833fe1.html" title="Last updated on Feb 15, 2022, 3:41 PM UTC (5833fe1)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -117,6 +117,10 @@ Relation of Process Document to Patent Policy</h2>
 	The <a href="https://www.w3.org/Consortium/Patent-Policy/">W3C Patent Policy</a> [[!PATENT-POLICY]]
 	is incorporated by normative reference as a part of the Process Document,
 	and is thus equally binding.
+	These policies are written with the intent that there be no conflict, 
+	but in the event that one exists, the Member Agreement takes precedence over this Process, 
+	which takes precedence over any other normative references. 
+	In matters dealing with patents, the Patent Policy takes precedence.
```
[the remark this answers is not in the document]
Comment (Hmgov) [reply index.bs:123]: islamic country says i fucking politic asian countrys alls terarist dont tollk to politic i fucking islamic allah the showing your licence fucking tollking politic fucking your allah

**Comment by plehegar** [comment]
This looks like a duplicate of #572 and that one seem to address the comment from @nigelmegitt . @wseltzer , ok to close this pull request and push #572 instead?

File: index.bs (hunk 1 above)
[the remark this answers is not in the document]
Comment (dwsinger) [reply index.bs:123]: I had the same comment – the last sentence is incomplete, it doesn't say over what.

Comment (TallTed) [inline index.bs:123]:
```suggestion
	In matters dealing with patents, the Patent Policy takes precedence over ___, while it is subordinate to ___.
```

**Comment by nigelmegitt** [comment]
> This looks like a duplicate of #572 and that one seem to address the comment from @nigelmegitt . @wseltzer , ok to close this pull request and push #572 instead?

My comment was that the change in #572 is misplaced: https://github.com/w3c/w3process/pull/572#issuecomment-925042556

File: index.bs (hunk 1 above)
[the remark this answers is not in the document]
Comment (FYI5050997) [reply index.bs:123]:
```suggestion
	In matters dealing with patents, the Patent Policy takes precedence.
```

[the remark this answers is not in the document]
Comment (FYI5050997) [reply index.bs:123]: [🌎]renee.brutcher@icloud.com
Screenshot_20221206-083832_One UI Home

**Review by Chiramate — APPROVED** [review]

**Pull request closed without merging**